.github/workflows/smoke.yml

```diff
@@ -13,15 +13,16 @@ jobs:
     strategy:
       matrix:
         chrome-channel: ['stable', 'ToT']
-        smoke-test-invert: [false, true]
+        smoke-test-shard: [1, 2]
       # e.g. if set 1 fails, continue with set 2 anyway
       fail-fast: false
     runs-on: ubuntu-latest
     env:
-      # The smokehouse tests run by job `test set 1`. `test set 2` will run the rest.
-      SMOKE_BATCH_1: a11y oopif pwa dbw redirects errors offline
-    # Job named e.g. "Chrome stable, test set 1".
-    name: Chrome ${{ matrix.chrome-channel }}, batch ${{ matrix.smoke-test-invert == false && '1' || '2' }}
+      # The total number of shards. Set dynamically when length of single matrix variable is
+      # computable. See https://github.community/t/get-length-of-strategy-matrix-or-get-all-matrix-options/18342
+      SHARD_TOTAL: 2
```

Comment:
this is unfortunately really annoying. You can [get the length](${{ strategy.job-total }} ) of the _entire_ matrix, but that's 4 here, not 2 (since it's `chrome-channel` × `smoke-test-shard`). There are ways you can do [fancy things](https://github.community/t/getting-object-length/17802/3) with `| jq '. | length'`, but you need a reference to the whole array first, so you need it to be the [output of another job](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#example-returning-a-json-object) (can't do it with `env` because they aren't available inside `matrix`...), at which point it all becomes much more verbose than just having a `2` here.

So... :/